src/test/java/wooteco/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,268 @@
+package wooteco.subway.line;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.AcceptanceTest;
+import wooteco.subway.line.dto.LineResponse;
+import wooteco.subway.line.repository.LineRepository;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("노선 관련 기능")
+@Transactional
+public class LineAcceptanceTest extends AcceptanceTest {
+
+    @Autowired
+    private LineRepository lineRepository;
+
+    @DisplayName("노선을 생성한다.")
+    @Test
+    void createLine() {
+        // given
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "분당선");
+        params.put("color", "bg-red-600");
+
+        // when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/lines")
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(response.header("Location")).isNotBlank();
+    }
+
+    @DisplayName("이미 존재하는 노선의 이름으로 생성 요청 시 BAD_REQUEST를 응답한다.")
+    @Test
+    void createLineWhenDuplicateLineName() {
+        // given
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "분당선");
+        params.put("color", "bg-red-600");
+
+        // when
+        RestAssured.given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/lines")
+                .then().log().all()
+                .extract();
+
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/lines")
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    @DisplayName("모든 노선을 조회한다.")
+    @Test
+    void getLines() {
+        /// given
+        Map<String, String> params1 = new HashMap<>();
```

Comment:
```suggestion
        Map<String, String> 분당선 = new HashMap<>();
```

테스트 코드도 유지보수해야 되는 코드입니다. 의미 없는 이름을 주지 마세요.
테스트 코드에 한글 변수명을 쓰면 좀더 읽기 쉽고 파악하기 좋을테니 한 번 적용해보아요